packages/platform-sdk-nano/src/services/wallet-discovery.ts

```diff
@@ -0,0 +1,22 @@
+import { Contracts } from "@arkecosystem/platform-sdk";
+
+import { deriveAccount, deriveLegacyAccount } from "./identity/helpers";
+
+export class WalletDiscovery implements Contracts.WalletDiscoveryService {
+	public async fromMnemonic(mnemonic: string): Promise<Contracts.AddressDiscoveryDataTransferObject[]> {
+		const [legacy, standard] = await Promise.all([deriveAccount(mnemonic), deriveLegacyAccount(mnemonic)]);
```

Comment:
On nault.cc, the bip39 account derivation also takes an account index:

![image](https://user-images.githubusercontent.com/1091900/119943703-aee0d500-bf93-11eb-80c0-5c2b4811ae09.png)

Is this handled later? I assume the index is included in the path?